From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Wed, 24 Jun 2020 14:28:11 +0200
Subject: ANDROID: GKI: remove CONFIG_EDAC from arm64 gki_defconfig

It's no longer a needed option due to upstream changes so remove it from
the defconfig for arm64.

Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: Id1723b7b5a59c53671df19dddbd07f16d13da16f
---
 arch/arm64/configs/gki_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -368,7 +368,6 @@ CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_TRIGGERS=y
-CONFIG_EDAC=y
 CONFIG_RTC_CLASS=y
 CONFIG_RTC_DRV_PL030=y
 CONFIG_RTC_DRV_PL031=y
